traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -203,17 +301,124 @@ func selectMaxLastUpdatedQuery(where string) string {
 	select max(last_updated) as t from last_deleted l where l.table_name='job') as res`
 }
 
+// Deprecated, only to be used with versions below 4.0
+const readQuery = `
+SELECT job.id,
+        'PURGE' AS keyword,
+		ttl_hr,
+		asset_url,
+		start_time,
+		u.username as createdBy,
+		ds.xml_id as dsId
+FROM job
+JOIN tm_user u ON job.job_user = u.id
+JOIN deliveryservice ds ON job.job_deliveryservice = ds.id
+`
+
+// Almost the same as readQuery, but returns appropriate values for API 4.0+
+const readQueryV40 = `
```

Comment:
We should probably call this V4 instead of V40, because we'll likely just update this query with new fields if they're ever added.